shared/modules/selectors.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  ALLOWED_SMART_TRANSACTIONS_CHAIN_IDS,
+  SKIP_STX_RPC_URL_CHECK_CHAIN_IDS,
+} from '../constants/smartTransactions';
+import { ENVIRONMENT } from '../../development/build/constants';
+import {
+  getCurrentChainId,
+  getCurrentNetwork,
+  accountSupportsSmartTx,
+} from '../../ui/selectors/selectors'; // TODO: Migrate shared selectors to this file.
+import { getNetworkNameByChainId } from './feature-flags';
+
+export const getSmartTransactionsOptInStatus = (
```

Comment:
We could do that, but we should anyway have a selectors.ts file to export all the shared selectors, because then we only need to do 1 import instead of importing e.g. 5 different selector files.